integration_tests/src/main/python/orc_test.py

```diff
@@ -45,9 +45,30 @@ def test_basic_read(std_input_path, name, read_func, v1_enabled_list, orc_impl,
             read_func(std_input_path + '/' + name),
             conf=all_confs)
 
-orc_gens_list = [[byte_gen, short_gen, int_gen, long_gen, float_gen, double_gen,
+orc_basic_gens = [byte_gen, short_gen, int_gen, long_gen, float_gen, double_gen,
     string_gen, boolean_gen, DateGen(start=date(1590, 1, 1)),
-    TimestampGen(start=datetime(1590, 1, 1, tzinfo=timezone.utc))],
+    TimestampGen(start=datetime(1590, 1, 1, tzinfo=timezone.utc))]
+
+# Set to true after the issue https://github.com/rapidsai/cudf/issues/8704 is fixed.
```

Comment:
We can't check this in as-is -- it can silently corrupt data on an ORC read.  Either we need to make the struct support qualified by a separate incompatible config that defaults to off or wait until it is fixed.

Reply:
The issue rapidsai/cudf#8704 is marked as  " v21.08 Release", so  I plan to wait for the fix.